tests/sentry/api/endpoints/test_organization_user_issues_search.py

```diff
@@ -2,53 +2,48 @@
 
 from six.moves.urllib.parse import urlencode
 
-from datetime import timedelta
 from django.core.urlresolvers import reverse
-from django.utils import timezone
 
-from sentry import tagstore
-from sentry.models import EventUser, OrganizationMemberTeam
-from sentry.testutils import APITestCase
+from sentry.models import OrganizationMemberTeam
+from sentry.testutils import APITestCase, SnubaTestCase
+from sentry.testutils.helpers.datetime import iso_format, before_now
 
 
-class OrganizationUserIssuesSearchTest(APITestCase):
+class OrganizationUserIssuesSearchTest(APITestCase, SnubaTestCase):
```

Comment:
Should this move to the snuba directory to live with the other snuba endpoint tests?

Reply:
All the test suites depend on and run Snuba now, but the Snuba tests are still scattered in both the Snuba and Backend groups. It's quite confusing since the tests for a single module are often split between two different test suites.

Now that SnubaTagStorage is the default tagstore, I wonder if it's actually worth moving tests out of the Snuba directory instead where no longer needed.